### PR TITLE
WebGLTextures: Set all pixel storage modes for CubeTexture.

### PR DESCRIPTION
**Pull request by 11zouzouzou**
Unable to set the correct unpackAlignment when cubeTexture is set to dataTexture. So when i set the dataTexture(dataType:RBG) to Cubetexture the system will warn me and not work.

To solve：set the correct unpackAlignment

### CODE REVIEW

File: src/renderers/webgl/WebGLTextures.js [hunk 1]
```diff
@@ -713,6 +713,8 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 		const isCompressed = ( texture && ( texture.isCompressedTexture || texture.image[ 0 ].isCompressedTexture ) );
 		const isDataTexture = ( texture.image[ 0 ] && texture.image[ 0 ].isDataTexture );
 
+		if ( texture.image[ 0 ] ) _gl.pixelStorei( _gl.UNPACK_ALIGNMENT, texture.image[ 0 ].unpackAlignment );
```
Comment (Mugen87) [inline src/renderers/webgl/WebGLTextures.js:716]: I'm not sure this is correct. Why don't you configure `unpackAlignment` on your instance of `CubeTexture`.

Comment (11zouzouzou) [reply src/renderers/webgl/WebGLTextures.js:716]: 这是一个习惯的问题。如果认为以传入的cubeTexture为准可以设置为cubeTexture的unpackAlignment(default:4)。这里我以真正传入数据dataTexture的unpackAlignment(defalut:1)为准.

Comment (Mugen87) [reply src/renderers/webgl/WebGLTextures.js:716]: Well, if the alignment for all textures of a cube texture is equal (at that is the default), `CubeTexture.unpackAlignment` should be configured. 

Just extracting properties from the first texture in the cube texture image array is not consistent. What is if the properties differ for subsequent textures?

Comment (11zouzouzou) [reply src/renderers/webgl/WebGLTextures.js:716]: We cannot control the incoming data, but we can refer to the incoming cubeTexture unpackAlignment. Okay? Better than having the user unable to set it?

Comment (Mugen87) [reply src/renderers/webgl/WebGLTextures.js]: I guess I would add the following two lines so `uploadCubeTexture()` is similar to `uploadTexture()`.

```js
_gl.pixelStorei( _gl.UNPACK_PREMULTIPLY_ALPHA_WEBGL, texture.premultiplyAlpha );
_gl.pixelStorei( _gl.UNPACK_ALIGNMENT, texture.unpackAlignment );
```

File: src/renderers/webgl/WebGLTextures.js [hunk 2]
```diff
@@ -713,6 +713,8 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 		const isCompressed = ( texture && ( texture.isCompressedTexture || texture.image[ 0 ].isCompressedTexture ) );
 		const isDataTexture = ( texture.image[ 0 ] && texture.image[ 0 ].isDataTexture );
 
+		if ( texture.image[ 0 ] ) _gl.pixelStorei( _gl.UNPACK_ALIGNMENT, texture.unpackAlignment );
```
Comment (Mugen87) [inline src/renderers/webgl/WebGLTextures.js:716]: I guess I would add the following two lines so `uploadCubeTexture()` is similar to `uploadTexture()`.

```js
_gl.pixelStorei( _gl.UNPACK_PREMULTIPLY_ALPHA_WEBGL, texture.premultiplyAlpha );
_gl.pixelStorei( _gl.UNPACK_ALIGNMENT, texture.unpackAlignment );
```

Comment (11zouzouzou) [reply src/renderers/webgl/WebGLTextures.js:716]: I think so, thanks.

**Comment by Mugen87** [comment]
NIT: Please move both two new lines right below the existing:
```js
_gl.pixelStorei( _gl.UNPACK_FLIP_Y_WEBGL, texture.flipY );
```
The all `pixelStorei()` calls in `uploadCubeTexture()` are grouped together.

**Comment by Mugen87** [comment]
@mrdoob Do you know if there was a specific reason for excluding the following lines when using `CubeTexture`?
```js
_gl.pixelStorei( _gl.UNPACK_PREMULTIPLY_ALPHA_WEBGL, texture.premultiplyAlpha );
_gl.pixelStorei( _gl.UNPACK_ALIGNMENT, texture.unpackAlignment );
```

**Comment by 11zouzouzou** [comment]
The unpackAlignment is needed; premultiplyAlpha is not sure

**Comment by 11zouzouzou** [comment]
UNPACK_PREMULTIPLY_ALPHA_WEBGL will have default setting. If not set it, it is the default setting

**Comment by mrdoob** [comment]
> @mrdoob Do you know if there was a specific reason for excluding the following lines when using `CubeTexture`?

Not that I know of.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**